apps/ios/GuideDogs/Assets/Localization/en-US.lproj/Localizable.strings

```diff
@@ -3278,8 +3310,18 @@
 "osm.tag.garden" = "Garden";
 
 /* Open Street Map term. An area of open space for recreational use, usually designed and in semi-natural state with grassy areas, trees and bushes. Parks are usually urban, but not always. */
-"osm.tag.park" = "Park";
-
+"osm.tag.park" = "leisure=park";
+
+//TODO JON
+"osm.tag.garden" = "leisure=garden";
+"osm.tag.playground" = "leisure=playground";
+"osm.tag.nature_reserve" = "leisure=nature_reserve";
+"osm.tag.botanical_garden" = "leisure=botanical_garden";
+"osm.tag.public_garden" = "leisure=public_garden";
+"osm.tag.field" = "leisure=field";
+"osm.tag.reserve" = "leisure=reserve";
+"osm.tag.green_space" = "leisure=green_space";
+"osm.tag.recreation_area" = "leisure=recreation_area";
```

Comment:
These should be the human-friendly version of the text, e.g. `"osm.tag.public_garden" = "Public Garden";`. This is what gets displayed in the UI when them entity doesn't have a name.